dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClasspathCheckOption.java

```diff
@@ -123,21 +129,30 @@ private static Options configureOptions() {
     return options;
   }
 
-  static ImmutableList<Path> generateInputClasspath(CommandLine commandLine)
-      throws RepositoryException, ParseException {
-    setRepositories(commandLine);
-
+  static ImmutableList<Artifact> generateArtifacts(CommandLine commandLine)
+      throws RepositoryException {
+    Preconditions.checkArgument(
+        commandLine.hasOption("b") || commandLine.hasOption("a"),
+        "The arguments must have option 'a' or 'b' to list Maven artifacts");
     if (commandLine.hasOption("b")) {
       String bomCoordinates = commandLine.getOptionValue("b");
       DefaultArtifact bomArtifact = new DefaultArtifact(bomCoordinates);
-      List<Artifact> artifactsInBom = RepositoryUtility.readBom(bomArtifact);
-      return ClassPathBuilder.artifactsToClasspath(artifactsInBom);
-    } else if (commandLine.hasOption("a")) {
+      return ImmutableList.copyOf(RepositoryUtility.readBom(bomArtifact));
+    } else {
+      // option 'a'
       String[] mavenCoordinatesOption = commandLine.getOptionValues("a");
-      ImmutableList<Artifact> artifacts =
-          Arrays.stream(mavenCoordinatesOption)
-              .map(DefaultArtifact::new)
-              .collect(toImmutableList());
+      return Arrays.stream(mavenCoordinatesOption)
+          .map(DefaultArtifact::new)
+          .collect(toImmutableList());
+    }
+  }
+
+  static ImmutableList<Path> generateInputClasspath(CommandLine commandLine)
```

Comment:
Similarly here: You're not generating it.

Reply:
Fixed.